Add support for Pmod-TC1

tfors opened the pull request:
Added support for Pmod-TC1 by starting from Pmod-ALS as example.

schelleg commented:
Thanks for submitting driver - give me a couple of days to verify (I have to buy the device!).

tfors commented:
No problem.

yunqu commented on Pynq-Z1/sdk/pmod_tc1/src/pmod_tc1.c:
```diff
@@ -0,0 +1,130 @@
+   */
+  u8 raw_data[4];
+  spi_transfer(SPI_BASEADDR, 4, raw_data, NULL);
+  u32 v = ( (raw_data[0] << 24) + (raw_data[1] << 16) + (raw_data[2] << 8) + raw_data[3] );
```
Hi @tfors 

Line 75 seems to exceed the 79 characters. Can you please adjust it? 

We will probably upload some guidelines later.

Thanks,

Rock

tfors replied:
This should be fixed now.  Thanks for pointing it out!

tfors commented:
Hi @beja65536,

Oops.  I modified the init file on my pynq to make it work and forgot to mod the one in the repo.  I've fixed that now and I've also added a legend and some axis labels to the plot in the example notebook.

Thanks for reviewing this.
Tom
